Pipelines/Scripts/publish-upm.ps1

```diff
@@ -33,7 +33,7 @@ try {
     }
 
     if ($Authenticate.IsPresent) {
-        npm install -g vsts-npm-auth
+        npm install -g vsts-npm-auth --registry https://registry.npmjs.com --always-auth false
```

Comment:
Ahh interesting! Were you finding that this command was looking in the private registry in the .npmrc? Good catch! I probably already had this installed in my tests 😅

Reply:
I am not sure whether it was looking in the private registry in .npmrc or .upmconfig.toml (I configured the latter to only include the publishing feed), but it was definitely not looking in the npmjs one.